samples/diagnostics/program-visualization/README.md

```diff
@@ -0,0 +1,22 @@
+# Visualization of Q# Programs #
```

Comment:
If you provide a metadata header here, then this sample will get onboarded to the samples browser at docs.microsoft.com/quantum. For example, the header at https://raw.githubusercontent.com/microsoft/Quantum/master/samples/getting-started/qrng/README.md may be a useful place to start:

```suggestion
---
page_type: sample
languages:
- qsharp
products:
- qdk
description: "This sample shows how to visualize the execution of Q# programs using Jupyter Notebook."
---

# Visualization of Q# Programs
```

As a side note, the samples browser doesn't support `#` at the end of headings, so I've suggested removing that here as well as below in the file.